### DISCUSSION TITLE
How to bump MacOS deployment target for Fastlane.swift

### DISCUSSION THREAD

**Discussion by Lontronix** [fastlane/fastlane#22022] [Q&A]
Hi, I was wondering if it's possible to bump the deployment target for the auto-generated Xcode project that is used to edit Fastlane swift files.

**Comment by geraldine198** [comment]
Hello, @Lontronix [Spotify Pie](https://www.spotify-pie.com)

Here is how you can bump the deployment target for the auto-generated Xcode project used to edit Fastlane Swift files:

Locate the Xcode Project: Find the temporary Xcode project generated by Fastlane. This project is typically located within your fastlane directory, often in a hidden subdirectory.

Open the Project in Xcode: Open this auto-generated Xcode project in Xcode.

Change the Deployment Target:

In Xcode, select the project in the Project Navigator.
Choose the target for which you want to change the deployment target.
Navigate to the "General" tab in the main editor.
Under "Deployment Info," set the "iOS Deployment Target" to your desired version.
Save Changes: Save your changes in Xcode.

Ensure Persistence:

Since the Xcode project is auto-generated and might be overwritten, make sure to apply these changes automatically.
To do this, you can create a script that changes the deployment target each time the project is generated. This script would be part of your Fastlane setup to ensure the deployment target is set correctly whenever the Xcode project is regenerated.

I hope this information help you.

Best Regards.
geraldine198
